eventuate-util-common/src/main/java/io/eventuate/util/common/StringUtils.java

```diff
@@ -0,0 +1,13 @@
+package io.eventuate.util.common;
+
+import java.nio.charset.Charset;
+
+public class StringUtils {
```

Comment:
This is not generic StringUtils.
It's part of the encoding/decoding for messages

Reply:
I thought that thing is pretty general and can be used somewhere else in future.
Ok, what is the best place for it? And what name do you prefer.